managed/services/qan/client.go

```diff
@@ -128,6 +128,16 @@ func (c *Client) QueryExists(ctx context.Context, serviceID, query string) error
 	return nil
 }
 
+// IsReady verifies that qan-api2 works.
+func (c *Client) IsReady(ctx context.Context) error {
+	_, err := c.qsc.HealthCheck(ctx, nil)
+	if err != nil {
+		return errors.WithStack(err)
```

Comment:
Do we need to print the stack here?

Reply:
Removed, together with errors package usage.